src/content/ui-api/kubectl-gs/template-cluster.md

```diff
@@ -99,6 +109,21 @@ It supports the following flags:
 
 - `--external-snat` - AWS CNI configuration to disable (is enabled by default) the [external source network address translation](https://docs.aws.amazon.com/eks/latest/userguide/external-snat.html).
 
+### GCP specific
+
+- `--gcp-project` - The Google Cloud Platform project where the cluster will be deployed.
+- `--region` - The Google Cloud Platform region where the cluster will be deployed.
+- `--gcp-failure-domains` - The Google Cloud Platform zones where the cluster's control-plane nodes will be deployed.
+- `--gcp-control-plane-sa-email` (optional) - The Google Cloud Platform Service Account which the control-plane nodes will use (default: "default").
+- `--gcp-control-plane-sa-scopes` (optional) - The Google Cloud Platform API scopes the control-plane will have access to (default: "https://www.googleapis.com/auth/compute")
+- `--gcp-machine-deployment-name` (optional) - The name of the MachineDeployment (default: "worker1")
+- `--gcp-machine-deployment-instance-type` (optional) - The Google Cloud Platform Instance Type for the worker node isntances (default: "n1-standard-2")
+- `--gcp-machine-deployment-failure-domain` (optional) - The Google Cloud Platform zones where the cluster's worker nodes will be deployed. (default: "europe-west6-a")
+- `--gcp-machine-deployment-replicas` (optional) - The number of worker nodes (default: 3)
+- `--gcp-machine-deployment-disk-size` (optional) - The worker node disk size in GB (default: 100)
```

Comment:
These all apply to the default nodepool that is created, right? I was wondering if we should mention that instead of "worker nodes", because other nodepools can be created later with other configuration values.

Reply:
Done